pages/guides/how_to_set_up_full_node.md

```diff
@@ -4,60 +4,128 @@
+### Step 6: Get Genesis and Update Config
+
+```bash
+curl https://dydx-rpc.lavenderfive.com/genesis | python3 -c 'import json,sys;print(json.dumps(json.load(sys.stdin)["result"]["genesis"], indent=2))' > $WORKDIR/config/genesis.json
+
+# For a more updated list of seed nodes, visit https://docs.dydx.exchange/network/resources#seed-nodes
+SEED_NODES=("ade4d8bc8cbe014af6ebdf3cb7b1e9ad36f412c0@seeds.polkachu.com:23856", 
+"65b740ee326c9260c30af1f044e9cda63c73f7c1@seeds.kingnodes.net:23856", 
+"f04a77b92d0d86725cdb2d6b7a7eb0eda8c27089@dydx-mainnet-seed.bwarelabs.com:36656",
+"20e1000e88125698264454a884812746c2eb4807@seeds.lavenderfive.com:23856",
+"c2c2fcb5e6e4755e06b83b499aff93e97282f8e8@tenderseed.ccvalidators.com:26401",
+"a9cae4047d5c34772442322b10ef5600d8e54900@dydx-mainnet-seednode.allthatnode.com:26656",
+"802607c6db8148b0c68c8a9ec1a86fd3ba606af6@64.227.38.88:26656",
+"ebc272824924ea1a27ea3183dd0b9ba713494f83@dydx-mainnet-seed.autostake.com:27366"
+)
 
-## Install Bware’s snapshot (optional but saves days)
+sed -i 's/seeds = ""/seeds = "'"${SEED_NODES[*]}"'"/' $HOME/.dydxprotocol/config/config.toml
+```
+
+### Step 7: Create Service to Start Full Node
+
+We will create a systemd service to start the full node.  This will allow the full node to start automatically on boot.
+
+```bash
+sudo tee /etc/systemd/system/dydxprotocold.service > /dev/null << EOF
+[Unit]
+Description=osmosis node service
+After=network-online.target
+
+[Service]
+User=$USER
+ExecStart=/$HOME/go/bin/cosmovisor run start --non-validating-full-node=true
+WorkingDirectory=$HOME/.dydxprotocol
+Restart=always
+RestartSec=5
+LimitNOFILE=4096
+Environment="DAEMON_HOME=$HOME/.dydxprotocol"
+Environment="DAEMON_NAME=dydxprotocold"
+Environment="DAEMON_ALLOW_DOWNLOAD_BINARIES=false"
+Environment="DAEMON_RESTART_AFTER_UPGRADE=true"
+Environment="UNSAFE_SKIP_BACKUP=true"
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+sudo systemctl daemon-reload
+sudo systemctl enable dydxprotocold
+
+```
+
+### Step 8: Get Snapshot
 > **Note:** the example values below align with the **deployment by DYDX token holders**. For alternatives, please visit the [Network Resources page](../network/resources.md).
 
 1. From https://bwarelabs.com/snapshots/dydx
-2. Download and extract (using `lz4 -dc < snapshotfile.tar.lz4 | tar xf -`) the snapshot contents in the data directory (make sure you are in the data directory before running the tar command).  Important: The home directory (`$DYDX_HOME` or `/home/vmware/.dydx-mainnet-1` in our example) contains another `data/` directory.
+2. Download and extract (using `lz4 -dc < snapshotfile.tar.lz4 | tar xf -`) the snapshot contents in the dydxprotocol home directory (make sure you are in this directory before running the tar command). In this example, the home directory is `$HOME/.dydxprotocol`
 3. (Alternatives): If the above is not available, there are these alternatives:
```

Comment:
link to [here](https://docs.dydx.exchange/network/resources#snapshot-service) instead